### ISSUE TITLE
Why is there a gap between the PSNR in paper and the results of official pre-trained model?

### ISSUE THREAD

**Issue by CJ0109** [XPixelGroup/BasicSR#634]
The following are the results of official pre-trained model EDSR_Lx4 (https://drive.google.com/drive/folders/1rtJCHuOAEixB1OWmUVbbVm158vzC3kTt)
![image](https://github.com/XPixelGroup/BasicSR/assets/41496128/1a27ae33-7eba-46d9-8729-b46021ecac9d)

and the following is reported in EDSR paper
![image](https://github.com/XPixelGroup/BasicSR/assets/41496128/0927412c-4dc9-44f4-b4a9-d74528241065)

For x4 SR, the model trained on DIV2K. The test results on Set5 and Set14 seem to be very different from those written in the paper, but the results on the DIV2K validation set are similar. Why is this? Are more pixels cropped when computing PSNR on Set5 and Set14?

**Comment by CJ0109** [comment]
test results/paper
Set5: 30.5582/32.46
Set14: 27.0147/28.80
DIV2K val: 29.2770/29.25

**Comment by hanzhangshen03** [comment]
In your test configuration (yml) file, you need to set `test_y_channel` to `true` under val/metrics. Hope this helps.

**Comment by YAN287-hub** [comment]
> 在您的测试配置（yml）文件中，需要在 val/metrics 下`test_y_channel`进行设置。希望这能帮到您。`true`

你好，请问我要训练单通道红外数据集配置文件该怎么设置呢  img_range: 255.
  rgb_mean: [0.4488, 0.4371, 0.4040]这两个参数
